src/ext/java/org/opentripplanner/ext/legacygraphqlapi/generated/LegacyGraphQLTypes.java

```diff
@@ -1544,7 +1544,7 @@ public static class LegacyGraphQLQueryTypePlanArgs {
     private Boolean ignoreRealtimeUpdates;
     private List<LegacyGraphQLInputCoordinatesInput> intermediatePlaces;
     private Double itineraryFiltering;
-    private Integer keepingRentedBicycleAtDestinationCost;
```

Comment:
Breaking changes to the API schema must be made very carefully as it breaks clients.

Basically, you need to revert the changes in this file.